### DISCUSSION TITLE
Не могу загрузить фото для подтверждения академического статуса

### DISCUSSION THREAD

**Discussion by FaCSM** [community/community#129875] [GitHub Education]
### Hi everyone,

Не могу загрузить фото для подтверждения академического статуса, фото формата .jpg сделано со смартфона, пытался использовать:
ПК:
Opera One(версия: 110.0.5130.82)
Система:Windows 11 64-bit
Версия Chromium:124.0.6367.243
и
Google Chrome
Версия 126.0.6478.116 (Официальная сборка), (64 бит)
Android:
Brave 1.67.119, Chromium 126.0.6478.114

**Comment by ebndev** [comment]
Hi @FaCSM, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
